template/template.go

```diff
@@ -76,3 +84,137 @@ func (t *Template) ExecuteHTMLString(html string, data interface{}) (string, err
 	err = tmpl.Execute(&buf, data)
 	return buf.String(), err
 }
+
+type FuncMap map[string]interface{}
+
+var DefaultFuncs = FuncMap{
+	"toUpper": strings.ToUpper,
+	"toLower": strings.ToLower,
+	"toTitle": strings.ToTitle,
```

Comment:
https://github.com/prometheus/prometheus/issues/1241 filed to get this updated in Prometheus